python/examples/jsonschema/pointcloud_csv_to_mcap.py

```diff
@@ -79,7 +79,7 @@ def main():
     # tutorial-write-header-start
     with open(args.output, "wb") as f:
         writer = Writer(f)
-        writer.start("x-jsonschema", library="my-excellent-library")
+        writer.start("", library="my-excellent-library")
```

Comment:
Imo the writer shouldn't be setting their own library string in 99% of cases either. The library they are using to write the mcap file is our python library, so it should be automatically included.

Reply:
Yep - this is better IMO.
```suggestion
        writer.start()
```